### PR TITLE
Refactor legacy content removal

### PR DESCRIPTION
**Pull request by Uberbagel**
Some legacy content that got missed last time. We should release a build in the next day or two to test our queued changes before release, then this should be merged for the public release build as it will kill save games.

### CODE REVIEW

**Comment by Darxo** [comment]
Some of that removal will end a long streak of save game compatibility. 
While it doesn't affect recent new games, it will affect ones from long ago which still have those perks present.

**Comment by Uberbagel** [comment]
I asked Midas and he said it wouldn't

**Comment by LordMidas** [comment]
It won't break savegames from the recent builds. We required a new run from 0.5.0 onwards and I believe since then these perks are not being used by players nor are they available to them - so it should be fine to remove them.

**Review by Darxo — APPROVED** [review]

File: unpacked_brushes/rf_ui/metadata.xml [hunk 1]
```diff
@@ -60,19 +60,16 @@
 	<sprite id="perk_rf_entrenched_mini" ic = "FF23165B" img="skills\mini\perk_rf_entrenched_mini.png" />
 	<sprite id="perk_rf_exploit_opening_mini" ic = "FF23165B" img="skills\mini\perk_rf_exploit_opening_mini.png" />
 	<sprite id="perk_rf_exude_confidence" ic = "FF23165B" img="..\..\gfx\ui\perks\perk_rf_exude_confidence.png" />
-	<sprite id="perk_rf_eyes_up_mini" ic = "FF23165B" img="skills\mini\perk_rf_eyes_up_mini.png" />
```
Comment (LordMidas) [inline unpacked_brushes/rf_ui/metadata.xml:63]: Please also remove the associated mini png file from the `unpacked_brushes\skills\mini` folder.

**Review by LordMidas — APPROVED** [review]

File: unpacked_brushes/rf_ui/metadata.xml (hunk 1 above)
Comment (Uberbagel) [reply unpacked_brushes/rf_ui/metadata.xml:63]: done

**Pull request merged**